media/js/firefox/new/experiment-firefox-new-batm-anim.js

```diff
@@ -0,0 +1,18 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+(function() {
+    'use strict';
+
+    var cop = new Mozilla.TrafficCop({
+        id: 'experiment_firefox_new_batm_anim',
+        variations: {
+            'xv=batmprivate': 33,   // Control: basic page, no machine
```

Comment:
Hmm, this is also going to cause an issue since (once you remove this line) it will redirect to `?xv=batmprivate&xv=batma`. This isn't something traffic cop is set up to do, as it was designed to display variations based on the original URL, not redirect to another URL entirely.